project/src/private-route/private-route.tsx

```diff
@@ -0,0 +1,25 @@
+import { Redirect, Route, RouteProps } from 'react-router-dom';
+import { AppRoutes, AuthorizationStatus } from '../consts';
+
+type PrivateRouteProps = RouteProps & {
+  render: () => JSX.Element;
+  authorizationStatus: AuthorizationStatus;
+};
+
+function PrivateRoute(props: PrivateRouteProps): JSX.Element {
+  const {exact, path, render, authorizationStatus} = props;
+
+  return (
+    <Route
+      exact = {exact}
```

Comment:
Здесь лучше без пробелов
```suggestion
      exact={exact}
```